### DISCUSSION TITLE
Field conditions - allow fields to optionally be inactive intead of being hidden

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#15903] [Ideas] [answered]
I think there could be situations where we want to inform admin user that field exists, but is just inactive due to some conditions not being met.

**Comment by ryanleichty** [comment]
Perhaps **read only** instead of **inactive**? [Sanity](https://www.sanity.io/docs/conditional-fields) has `hidden` and `readOnly` field options. Shopify's plugin for Sanity does this where they'll display your storefront data directly within your Sanity Studio as read only. That way you can view your Shopify data alongside your CMS data.

**Comment by brandonkelly** [comment, marked as answer]
Added for 5.7! #16805

**Reply by brandonkelly** [reply]
Craft 5.7.0 is out with that change.
